v20.2/alter-column.md

```diff
@@ -59,6 +59,8 @@ For an example of `ALTER COLUMN TYPE`, see [Increase a column type's precision](
 
 ## Examples
 
+>ALTER TABLE student ALTER clg_fee type DECIMAL (12,2);
```

Comment:
All example code snippets must be properly formatted.

The code snippet you provided should look like this:

```
{% include copy-clipboard.html %}
~~~ sql
> ALTER TABLE student ALTER clg_fee type DECIMAL (12,2);
~~~
```

For reference, see the first code snippet in the example with the title `### Set or change a DEFAULT value`:

```
{% include copy-clipboard.html %}
~~~ sql
> ALTER TABLE subscriptions ALTER COLUMN newsletter SET DEFAULT true;
~~~
```